app/javascript/components/Results/ResultGrid/ResultGrid.tsx

```diff
@@ -56,7 +58,9 @@ export const ResultGrid = ({ result, affiliate, query, position, vertical }: Res
 
   return (
     <GridContainer className='result search-result-item'>
-      <Grid row gap="md">
+      <ResultGridWrapper
```

Comment:
`<ResultGridWrapper>` component can be imported at other modules like Jobs, Video, RSS etc for clickTracking implementation when whole `div` is clicked.
@lorawoodford